src/logmask.h

```diff
@@ -14,6 +14,14 @@
 #include <bitset>
 #include "log_levels.h"
 
+/*
+ * the test suite Coveralls: https://coveralls.io
+ * will complains because these functions are only
+ * used at developement time.
+ * Don't worry about unsed lines for the functions:
+ *   logmask_test
+ *   logmask_set_all
```

Comment:
Similarly, it seems like both of these should be exercised?

Reply:
Okay, I see you used the PP to hide the devel-only stuff :slightly_smiling_face:. However, these are not hidden, and I believe they *are* being exercised. Thus, is this comment (re: `logmask_test` and `logmask_set_all`) correct? I think it is not?